### ISSUE TITLE
Pre-work Checklist: Developer: Adaugo Akaluso

### ISSUE THREAD

**Issue by Adaakal** [hackforla/website#3782]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by Adaakal** [comment]
Provide Update
1. Progress: Reading up on Figma
2. Blockers: Getting organized with time.
3. Availability: This week: MTW 2/6-2/8 between 1-3
4. ETA: End of Weel

**Comment by Adaakal** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
This took about 4 weeks.

**Comment by ExperimentsInHonesty** [comment]
@Adaakal I am going to keep this open until you have done a full progress update on an issue (your last one just included progress).  It might not happen until the issue you are on gets to medium or large.

```
Provide Update
1. Progress
2. Blockers
3. Availability
4. ETA
```

**Comment by Adaakal** [comment]
@ExperimentsInHonesty that makes sense.

**Comment by Adaakal** [comment]
> **1. Progress: "What is the current status of your project? What have you completed and what is left to do?"**
Working on a small issue. 
> **2. Blockers: "Difficulties or errors encountered."**
None. Just need to have a big enough issue to have a well-rounded progress report.
> **3. Availability: "How much time will you have this week to work on this issue?"**
Varies
> **4. ETA: "When do you expect this issue to be completed?"**
When I have a medium or large issue 
> **5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."**
N/A

**Comment by ExperimentsInHonesty** [comment]
@Adaakal 

There are now

1 [back end Medium size issues ](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22+label%3A%22complexity%3A+medium%22+-label%3A%22role%3A+dev+leads%22#column-7198257)

AND 

2 [front end Medium size issues](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+medium%22+label%3A%22role%3A+front+end%22#column-7198257)

Available.  

That may change by the time you are ready to work on them, and we are always working on new ones.  

Please pick an issue from this link when you are ready to work on it.  If there are no issues at the link, when you are ready, write us back here, and I will keep an eye out for you again.  Please don't pick up an issue unless you will be working on it in 24 hours or less.

**Comment by Adaakal** [comment]
> 

Thanks Bonnie! 
Just confirming this statement: "Please don't pick up an issue unless you will be working on it in 24 hours or less."

You're saying that I should at least begin working on issues that I pick up within 24 hours. You are not saying that I should be done within 24 hours.

**Comment by jdingeman** [comment]
Hi @Adaakal - please provide us with an update on your overall progress when you can. Thanks!

**Comment by Adaakal** [comment]
Hi @jdingeman, I have to take a step back from HFLA because I'm starting classes again. I'm wondering where I can find the rules for people who can't be consistent with their involvement in HFLA. I think I read somewhere that we can still be involved in some way.

**Comment by roslynwythe** [comment]
Hi @Adaakal please let us know that you wish to continue as part of the HfLA team, by posting an update at least every two weeks.  Please refer to this https://github.com/hackforla/website/issues/3782#issuecomment-1545288631 for instructions on how to add an update.  Thanks!

**Comment by Adaakal** [comment]
Progress: "What is the current status of your project? What have you completed and what is left to do?"
I would like to continue as part of the HFLA team. I would like to help in ways that don't require the 6 hour/week minimum
Blockers: "Difficulties or errors encountered."
Starting a Master's program
Availability: "How much time will you have this week to work on this issue?"
It will vary each week
ETA: "When do you expect this issue to be completed?"
Pictures (optional): "Add any pictures of the visual changes made to the site so far."

**Comment by roslynwythe** [comment]
Hi @Adaakal If you would like to continue with Hack for LA, please leave an update at least every two weeks.  I see that you mentioned you will be starting a Masters program.  Do you think you will have time to contribute to Hack for LA during summer break?  If not, we could close this issue and you would be welcome to come back in the future and reopen it.  If we don't hear from you by 6/19/2023 we will go ahead and close it.

**Comment by Adaakal** [comment]
Hi @roslynwythe, I would like to stay active. I'm currently in the Summer Semester, but I still check the backlog weekly. Is it possible for me to go between the website team and design team and look for projects that aren't too high of a priority at the moment?

**Comment by Adaakal** [comment]
Continuing to check for issues sI can start within 24 hours.

**Comment by roslynwythe** [comment]
Hi @Adaakal I hope that you are well.  If you have time to jump on a time-sensitive front-end medium complexity issue, we do have at least one.  
[front end medium issues](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22complexity%3A+medium%22#column-7198257)

**Comment by t-will-gillis** [comment]
Inactive member; closed due to inactivity

**Comment by HackforLABot** [comment]
## Activity Log: Adaakal
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- Adaakal opened: https://github.com/hackforla/website/issues/3782 at 01/03/2023, 08:46 PM PST
- Adaakal assigned: https://github.com/hackforla/website/issues/3782 at 01/04/2023, 08:24 PM PST
- Adaakal assigned: https://github.com/hackforla/website/issues/2863#issuecomment-1048465499 at 01/05/2023, 07:57 PM PST
- Adaakal commented on: https://github.com/hackforla/website/issues/2863#issuecomment-1373113594 at 01/05/2023, 08:01 PM PST
- Adaakal opened: https://github.com/hackforla/website/pull/3798 at 01/06/2023, 03:23 PM PST
- Adaakal commented on: https://github.com/hackforla/website/pull/3798#issuecomment-1377593580 at 01/10/2023, 09:17 AM PST
- Adaakal PR was merged: https://github.com/hackforla/website/pull/3798#event-8196905286 at 01/10/2023, 04:07 PM PST
- Adaakal issue closed by PR 3798: https://github.com/hackforla/website/issues/2863#event-8196905380 at 01/10/2023, 04:07 PM PST
- Adaakal commented on: https://github.com/hackforla/website/pull/3798#issuecomment-1386415123 at 01/17/2023, 07:08 PM PST
- Adaakal assigned: https://github.com/hackforla/website/issues/1478#issuecomment-829718468 at 01/17/2023, 07:30 PM PST
- Adaakal unassigned: https://github.com/hackforla/website/issues/1478#issuecomment-1386434200 at 01/17/2023, 08:21 PM PST
- Adaakal assigned: https://github.com/hackforla/website/issues/3540#issuecomment-1244594704 at 01/17/2023, 08:22 PM PST
- Adaakal commented on: https://github.com/hackforla/website/issues/3540#issuecomment-1386470129 at 01/17/2023, 08:27 PM PST
- Adaakal commented on: https://github.com/hackforla/website/issues/3540#issuecomment-1397449627 at 01/19/2023, 10:49 AM PST
- Adaakal commented on: https://github.com/hackforla/website/pull/3811#issuecomment-1397452563 at 01/19/2023, 10:52 AM PST
- Adaakal commented on: https://github.com/hackforla/website/pull/3811#issuecomment-1397454619 at 01/19/2023, 10:54 AM PST
- Adaakal submitted review: https://github.com/hackforla/website/pull/3811#pullrequestreview-1262515023 at 01/19/2023, 11:44 AM PST
- Adaakal unassigned: https://github.com/hackforla/website/issues/3540#issuecomment-1397449627 at 01/22/2023, 10:14 AM PST
- Adaakal assigned: https://github.com/hackforla/website/issues/3383 at 01/22/2023, 10:17 AM PST
- Adaakal commented on: https://github.com/hackforla/website/issues/3383#issuecomment-1407507173 at 01/28/2023, 03:04 PM PST
- Adaakal opened: https://github.com/hackforla/website/pull/3888 at 01/28/2023, 07:00 PM PST
- Adaakal commented on: https://github.com/hackforla/website/pull/3888#issuecomment-1415119370 at 02/02/2023, 10:32 PM PST
- Adaakal commented on: https://github.com/hackforla/website/pull/3883#issuecomment-1416328238 at 02/03/2023, 11:47 AM PST
- Adaakal PR was merged: https://github.com/hackforla/website/pull/3888#event-8437337771 at 02/03/2023, 04:31 PM PST
- Adaakal issue closed by PR 3888: https://github.com/hackforla/website/issues/3383#event-8437337856 at 02/03/2023, 04:31 PM PST
- Adaakal assigned: https://github.com/hackforla/website/issues/3274#issuecomment-1290249941 at 02/06/2023, 01:34 PM PST
- Adaakal commented on: https://github.com/hackforla/website/issues/3274#issuecomment-1419800533 at 02/06/2023, 01:39 PM PST
- Adaakal commented on: https://github.com/hackforla/website/issues/3782#issuecomment-1419816549 at 02/06/2023, 01:49 PM PST
- Adaakal commented on: https://github.com/hackforla/website/issues/3782#issuecomment-1419818118 at 02/06/2023, 01:50 PM PST
- Adaakal closed as completed: https://github.com/hackforla/website/issues/3782#event-8452150061 at 02/06/2023, 01:50 PM PST
- Adaakal commented on: https://github.com/hackforla/website/issues/3274#issuecomment-1426308459 at 02/10/2023, 12:29 PM PST
- Adaakal commented on: https://github.com/hackforla/website/issues/3598#issuecomment-1426387958 at 02/10/2023, 01:52 PM PST
- Adaakal commented on: https://github.com/hackforla/website/issues/3274#issuecomment-1426400372 at 02/10/2023, 02:05 PM PST
- Adaakal unassigned: https://github.com/hackforla/website/issues/3274#issuecomment-1426400372 at 02/12/2023, 10:28 AM PST
- Adaakal assigned: https://github.com/hackforla/website/issues/3118#issuecomment-1421940683 at 02/12/2023, 11:06 AM PST
- Adaakal commented on: https://github.com/hackforla/website/issues/3118#issuecomment-1427109306 at 02/12/2023, 11:09 AM PST
- Adaakal commented on: https://github.com/hackforla/website/issues/3782#issuecomment-1430539973 at 02/14/2023, 03:47 PM PST
- Adaakal commented on: https://github.com/hackforla/website/issues/3118#issuecomment-1434031305 at 02/16/2023, 07:03 PM PST
- Adaakal opened: https://github.com/hackforla/website/pull/4017 at 02/20/2023, 03:06 PM PST
- Adaakal commented on: https://github.com/hackforla/website/pull/4017#issuecomment-1438935497 at 02/21/2023, 10:40 AM PST
- Adaakal commented on: https://github.com/hackforla/website/pull/4017#issuecomment-1442802112 at 02/23/2023, 08:43 PM PST
- Adaakal commented on: https://github.com/hackforla/website/issues/3782#issuecomment-1444329914 at 02/24/2023, 11:35 AM PST
- Adaakal commented on: https://github.com/hackforla/website/pull/4017#issuecomment-1449278579 at 02/28/2023, 07:28 PM PST
- Adaakal commented on: https://github.com/hackforla/website/pull/4017#issuecomment-1458554557 at 03/07/2023, 09:25 AM PST
- Adaakal commented on: https://github.com/hackforla/website/pull/4017#issuecomment-1458673296 at 03/07/2023, 11:00 AM PST
- Adaakal PR was merged: https://github.com/hackforla/website/pull/4017#event-8737227490 at 03/13/2023, 03:50 PM PDT
- Adaakal issue closed by PR 4017: https://github.com/hackforla/website/issues/3118#event-8737227643 at 03/13/2023, 03:50 PM PDT
- Adaakal assigned: https://github.com/hackforla/website/issues/4159 at 03/24/2023, 05:02 AM PDT
- Adaakal assigned: https://github.com/hackforla/website/issues/4239 at 03/24/2023, 05:03 AM PDT
- Adaakal commented on: https://github.com/hackforla/website/issues/4239#issuecomment-1482695631 at 03/24/2023, 05:05 AM PDT
- Adaakal unassigned: https://github.com/hackforla/website/issues/4239 at 03/26/2023, 04:39 PM PDT
- Adaakal unassigned: https://github.com/hackforla/website/issues/4159 at 03/26/2023, 04:41 PM PDT
- Adaakal commented on: https://github.com/hackforla/website/issues/3782#issuecomment-1485746615 at 03/27/2023, 12:29 PM PDT
- Adaakal assigned: https://github.com/hackforla/website/issues/4226 at 03/27/2023, 12:36 PM PDT
- Adaakal commented on: https://github.com/hackforla/website/issues/4226#issuecomment-1485763134 at 03/27/2023, 12:43 PM PDT
- Adaakal opened: https://github.com/hackforla/website/pull/4297 at 03/27/2023, 04:22 PM PDT
- Adaakal commented on: https://github.com/hackforla/website/pull/4297#issuecomment-1490842390 at 03/30/2023, 12:42 PM PDT
- Adaakal commented on: https://github.com/hackforla/website/issues/4226#issuecomment-1490855113 at 03/30/2023, 12:50 PM PDT
- Adaakal commented on: https://github.com/hackforla/website/issues/4226#issuecomment-1495052986 at 04/03/2023, 03:11 PM PDT
- Adaakal unassigned: https://github.com/hackforla/website/issues/4226 at 04/03/2023, 03:11 PM PDT
- Adaakal PR was closed: https://github.com/hackforla/website/pull/4297#event-8991995531 at 04/12/2023, 08:03 PM PDT
- Adaakal commented on: https://github.com/hackforla/website/issues/3782#issuecomment-1520607115 at 04/24/2023, 11:04 AM PDT
- Adaakal commented on: https://github.com/hackforla/website/issues/3782#issuecomment-1549713160 at 05/16/2023, 06:49 AM PDT
